packages/nextjs/app/builders/0x558d82B022a8A42E9684BaCaA97a35318d4C293A/page.tsx

```diff
@@ -0,0 +1,90 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const BagusinoPage: NextPage = () => {
+  return (
+    <div className="w-full h-screen bg-gradient-to-br from-blue-300 via-white to-gray-400 dark:from-gray-900 dark:via-black dark:to-gray-700 transition-colors duration-500 flex justify-center items-center">
+      <div className="relative w-[90%] max-w-[600px] h-auto bg-white dark:bg-gray-800 rounded-xl shadow-2xl overflow-hidden">
+        {/* Card Header */}
+        <div className="flex justify-center mt-4">
+          <div className="w-32 h-32 md:w-32 md:h-32 border-4 border-gray-800 dark:border-white rounded-full overflow-hidden">
+            <Image
+              src="https://avatars.githubusercontent.com/u/15916181?v=4"
+              alt="Profile Picture"
+              width={128}
+              height={128}
+              className="object-cover w-full h-full"
+            />
+          </div>
+        </div>
+        {/* Card Content */}
+        <div className="p-4 pt-2 md:p-2 text-center">
+          <h1 className="text-xl md:text-2xl font-bold text-gray-800 dark:text-gray-200 m-1">Bagusino</h1>
+          <p className="text-sm md:text-md text-gray-600 dark:text-gray-200 m-0">Multiplatform Developer</p>
+          <div className="flex justify-center mt-2 space-x-2">
+            <Address address="0x558d82B022a8A42E9684BaCaA97a35318d4C293A" format="short" />
+          </div>
+        </div>
+        {/* Footer */}
+        <div className="p-6 pt-2">
+          <div className="text-md text-gray-600 dark:text-gray-200 relative group ">
+            Born in the 90s in Spain, I{"'"}ve been working as multiplatform developer for 6 years. I am currently
+            finishing my final degree project in Computer Engineering and my master{"'"}s thesis in Cybersecurity. I
+            work as a Cybersecurity Consultant for EAL6 products for{" "}
+            <div
+              className="tooltip tooltip-primary dark:tooltip-primary"
+              data-tip="Look for your physical cold wallet Certifications 👀"
+            >
+              <div className="text-black dark:text-white font-bold group-hover:underline"> Common Criteria </div>
+            </div>
+            Certification in international markets. I am passionate about Blockchain and its potential to change the

```

Comment:
There is no space between "Criteria" and "Certification." This happens because, in JSX, white spaces between elements are ignored. You can add a space like this:  

```jsx
<span>Item 1</span>{" "}
<span>Item 2</span>
```